R/ResampleDesc.R

```diff
@@ -49,12 +49,10 @@
 #'   \item{folds [\code{integer(1)}]}{Folds in the repeated CV for \code{RepCV}.
 #'     Here \code{iters = folds * reps}. Default is 10.}
 #'   \item{horizon [\code{integer(1)}]}{Number of observations to forecast for \code{GrowthCV}
-#'    and \code{FixedCV}.}
-#'   \item{initial.window [\code{integer(1)}]}{Initial number of observations to start with
-#'    in \code{GrowthCV} and \code{FixedCV}.}
-#'   \item{size [\code{integer(1)}]}{Number of observations in data set. Used for \code{GrowthCV}
-#'    and \code{FixedCV}.}
-#'   \item{skip [\code{integer(1)}]}{ Number of cross validations to skip in \code{GrowthCV} and \code{FixedCV}.}
+#'    and \code{FixedCV}. default is 1}
```

Comment:
Three thoughts:
(1) the default is actually 0 as things stand
(2) maybe it would be better to rename it "step size" and make the default *actually* 1?
(3) maybe it would be useful to have the step size default to the `size` (prediction horizon size) value?